.github/workflows/publish-to-github-pages.yml

```diff
@@ -12,6 +12,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Install rsync
+        run: sudo apt-get update && sudo apt-get install -y rsync
```

Comment:
Just wondering why the rsync was needed here?

Reply:
There is a note [here](https://github.com/JamesIves/github-pages-deploy-action#using-a-container-) stating that if you are using the action on a container (instead of using it from JavaScript), you must install rsync prior to running the action. 

I've tested the action without installing rsync and it failed.